Sources/FoundationEssentials/String/String+Internals.swift

```diff
@@ -19,6 +19,61 @@ internal import Foundation_Private.NSString
 import Darwin
 #endif
 
+#if os(Windows)
+import WinSDK
+
+extension String {
+    package func withNTPathRepresentation<Result>(_ body: (UnsafePointer<WCHAR>) throws -> Result) throws -> Result {
+        guard !isEmpty else {
+            throw CocoaError.error(.fileReadInvalidFileName, userInfo: [NSFilePathErrorKey:""])
+        }
+
+        // 1. Normalize the path first.
+
+        var path = self
+
+        // Strip the leading `/` on a RFC8089 path (`/[drive-letter]:/...` ).  A
+        // leading slash indicates a rooted path on the drive for teh current
```

Comment:
I believe I will make more valuable comment one day, but today is not that day. Today is nitpicking only:
```suggestion
        // leading slash indicates a rooted path on the drive for the current
```